app/views/helpers/inputSelect.scala.html

```diff
@@ -43,7 +43,7 @@
             <option>@{args.toMap.get('_emptyValueText)}</option>
         }
         @options.map { v =>
-            <option id="@(id)-@{v.replaceAll(" ", "-")}" value="@v" @if(value.exists(_.toLowerCase == v)){ selected } >@Messages(s"$id.${v.replaceAll(" ", "")}")</option>
+            <option id="@(id)-@{v.replaceAll(" ", "-")}" value="@v" @if(value.exists(_.equalsIgnoreCase(v))){ selected } >@Messages(s"$id.${v.replaceAll(" ", "")}")</option>
```

Comment:
is there a hMRC inputselect component we can use that already does this?

Reply:
I am looking at uk.gov.hmrc.play.views.html.helpers._  I see input and inputRadioGroup but not inputSelect